symplyphysics/laws/thermodynamics/van_der_waals_critical_parameters.py

```diff
@@ -0,0 +1,72 @@
+from sympy import Eq
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+)
+from symplyphysics.core.quantity_decorator import validate_multiple_output
+
+# Description
+## Critical parameters of the van der Waals equation of state are such value of volume, pressure, and
+## temperature at which the isotherm has an inflection point whose tangent at that point is zero, i.e.
+## the first and second derivatives of pressure with respect to volume at constant temperature are zero.
+
+# Law: V_c = 3 * b
+#      p_c = a / (27 * b**2)
+#      T_c = (8 * a) / (27 * R * b)
```

Comment:
Why not separate to 3 laws?

Reply:
To me it seemed to make more sense to group them together into one law, since all of them together describe the critical point of vdW fluids.

Reply:
Also, maybe make a separate folder for the vdW equation? There seems to be a lot of laws related to it

Reply:
Sure you can move laws here and there to make them more relevant to each other.

Reply:
As you see, grouping multiple laws in a single file resulted in worse code. You had to add new `validate` functions and break usual law structure.

Reply:
Yeah, I see. I'll break them apart

Reply:
Although looking at what Zulusssss does in #597 this multiple-results validate function might come in handy.

Reply:
Maybe. But we will postpone it until gain from new function will be obvious :)